src/test/java/org/cactoos/scalar/ScalarEnvelopeTest.java

```diff
@@ -45,6 +47,22 @@ public void envelopeDelegatesCalls() {
         ).affirm();
     }
 
+    @Test
+    public void propagatesException() {
+        new Assertion<>(
+            "must not alter the exception thrown by original Scalar",
+            new ScalarEnvelope<Integer>(
+                () -> {
+                    throw new UnsupportedOperationException("ok");
```

Comment:
@andreoss I'm sorry, but I haven't see this "ok" twice. Can you move them to a single local variable?

Reply:
@fabriciofx Done.